### PR TITLE
SF-3242: Add job convention to job model

### PR DESCRIPTION
**Pull request by rklind**

### CODE REVIEW

File: src/sdk/models/jobs.ts [hunk 1]
```diff
@@ -58,6 +58,10 @@ export interface JobSchedule {
 	max_execution?: TimeInterval;
 }
 
+export interface JobConvention {
+	match?: string;
```
Comment (jtirjan) [inline src/sdk/models/jobs.ts]: Should `match` be an enum?

File: src/sdk/models/jobs.ts [hunk 2]
```diff
@@ -58,6 +58,16 @@ export interface JobSchedule {
...
 }
 
+export enum JobConventionMatch {
+	Account = 'account',
+	Container = 'container',
+	Ldap = 'ldap'
+}
+
+export interface JobConvention {
+	match?: JobConventionMatch;
```
Comment (smbecker) [inline src/sdk/models/jobs.ts]: `match` is not a base property of convention options. If anything you would want `schedule` here.

Comment (rklind) [reply src/sdk/models/jobs.ts]: i may not be understanding your comment.  can we touch base on this?  i was referencing the  documentation [here](https://skysync.atlassian.net/wiki/spaces/SSP/pages/13959311/User-Drive+Mapping+Feature).  also, my testing seemed to be working properly.

Comment (rklind) [reply src/sdk/models/jobs.ts]: schedule for convention jobs is inside the transfer block.  inside the convention block is match, maby_by, remove_invalid, unmapped_policy, etc.

Comment (smbecker) [reply src/sdk/models/jobs.ts]: but those are specific to the type of convention job. For example, unmapped_policy does not apply to folder mapping.

Comment (rklind) [reply src/sdk/models/jobs.ts]: thanks for the chat @smbecker.  i get it now.  :)

**Pull request merged**